specviz/data/qt/ui/model_fitting_plugin.ui

```diff
@@ -0,0 +1,335 @@
+         <item>
+          <widget class="QLineEdit" name="line_edit_model_arithmetic"/>
+         </item>
+        </layout>
+       </widget>
+      </item>
+     </layout>
+    </widget>
+   </item>
+   <item>
+    <widget class="QGroupBox" name="group_box_fitting">
+     <property name="title">
+      <string>Fitting</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_3">
+      <item>
+       <widget class="QComboBox" name="combo_box_fitting"/>
+      </item>
+      <item>
+       <widget class="QCheckBox" name="check_box_advanced_settings">
+        <property name="styleSheet">
+         <string notr="true">.QCheckBox::indicator:unchecked {
+    image: url(:/icons/icons8-Forward-96.png);
+	height: 16px;
+	width: 16px;
+}
+
+.QCheckBox::indicator:unchecked:pressed {
+    image: url(:/icons/icons8-Forward-96.png);
+	height: 16px;
+	width: 16px;
+}
+
+.QCheckBox::indicator:checked {
+    image: url(:/icons/icons8-Expand Arrow-96.png);
+	height: 20px;
+	width: 20px;
+}
+
+.QCheckBox::indicator:checked:pressed {
+    image: url(:/icons/icons8-Expand Arrow-96.png);
+	height: 20px;
+	width: 20px;
+}</string>
+        </property>
+        <property name="text">
+         <string>Advanced settings</string>
+        </property>
+       </widget>
+      </item>
+      <item>
+       <widget class="QGroupBox" name="group_box_advanced_settings">
+        <property name="enabled">
+         <bool>true</bool>
+        </property>
+        <property name="checked">
+         <bool>false</bool>
+        </property>
+        <layout class="QFormLayout" name="formLayout">
+         <item row="0" column="0">
+          <widget class="QLabel" name="max_iterations_label">
+           <property name="text">
+            <string>Max Iterations</string>
+           </property>
+          </widget>
+         </item>
+         <item row="0" column="1">
+          <widget class="QLineEdit" name="max_iterations_line_edit">
+           <property name="text">
+            <string>100</string>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="0">
+          <widget class="QLabel" name="relative_error_label">
+           <property name="text">
+            <string>Relative Error</string>
+           </property>
+          </widget>
+         </item>
+         <item row="1" column="1">
+          <widget class="QLineEdit" name="relative_error_line_edit">
+           <property name="text">
+            <string>1e-07</string>
+           </property>
+          </widget>
+         </item>
+         <item row="2" column="0">
+          <widget class="QLabel" name="epsilon_label">
+           <property name="text">
+            <string>Epsilon</string>
+           </property>
+          </widget>
+         </item>
+         <item row="2" column="1">
+          <widget class="QLineEdit" name="epsilon_line_edit">
+           <property name="text">
+            <string>1.4901161193847656e-08</string>
```

Comment:
The LineEdit shows the end characters of the number, so when I first look at the "Advanced settings" tab it appears to show 161193847656e-08 as the Epsilon value and the full value isn't displayed unless you specific left arrow from inside the box.  You may want to have it show the starting characters or drop some decimal points to avoid confusion.